### DISCUSSION TITLE
Scrypted Stream encoding broken

### DISCUSSION THREAD

**Discussion by xOnlyFadi** [koush/scrypted#1298] [Q&A] [answered]
Hi I don't if it's the right way to talk about issues for scrypted nvr using ubuntu 22.04 from proxmox ve having problems on rtsp or webrtc making image look like this corrupted
![image](https://github.com/koush/scrypted/assets/63213308/2319da55-6794-4043-9d5e-89f815dc2e3b)

**Comment by koush** [comment]
what camera plugin

**Reply by xOnlyFadi** [reply]
Wyze

**Reply by koush** [reply]
can you try selecting the low stream in the nvr app

**Reply by xOnlyFadi** [reply]
I can try when im at home

**Reply by xOnlyFadi** [reply]
it works on low quality what is the reason that high quality encoding breaks?

**Reply by koush** [reply, marked as answer]
insufficient wifi bandwidth probably.

**Reply by xOnlyFadi** [reply]
i had wyze docker before that using high quality (1920x1080) this problem didn't happen is it because it uses different decoder because when i restart the wyze script on scrypted video become normal and then becomes like this after 1min

**Reply by xOnlyFadi** [reply]
@koush is it probably why the video is like this?

**Reply by koush** [reply]
for what it's worth, if wyze docker bridge was working, you can use that. I'm not confident I have the wyze plugin working correctly/optimally.

**Reply by xOnlyFadi** [reply]
I can enable wyze docker and then stream it to scrypted using rtsp plugin right?

**Reply by koush** [reply]
right

**Reply by xOnlyFadi** [reply]
Thanks and have a nice day

**Reply by xOnlyFadi** [reply]
turn out i have the same problem on low stream i appears for 5 seconds and then disappears and then reappear every 2 mins i don't what happening with the wyze plugin on scrypted maybe there will be a fix in the future for the plugin
